src/client/handlers/VaultsSecretsRemove.ts

```diff
@@ -30,21 +30,22 @@ class VaultsSecretsDelete extends UnaryHandler<
       );
       const vaultId =
         vaultIdFromName ?? vaultsUtils.decodeVaultId(input.nameOrId);
-      if (vaultId == null) {
-        throw new vaultsErrors.ErrorVaultsVaultUndefined();
-      }
+      if (vaultId == null) throw new vaultsErrors.ErrorVaultsVaultUndefined();
```

Comment:
You also need to validate that the `input.secretNames` is an array of strings that match the path format.

Reply:
No other command does any checking for ensuring the `secretName` (or any other relevant parameter) matches the path format or validating the type. I have specified types in the actual parameter, so those types should be upheld.

I should be able to add type checking, but I will probably need to write a utility function for validation of the path formatting, and I'm not sure we should be doing that for this PR.